.github/workflows/testcontainers-java.yml

```diff
@@ -28,12 +28,17 @@ concurrency:
 
 jobs:
   test:
-    name: "CrateDB: ${{ matrix.cratedb-version }}
+    name: "
+     Java: ${{ matrix.java-version }}
+     CrateDB: ${{ matrix.cratedb-version }}
      on ${{ matrix.os }}"
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [ "ubuntu-latest" ]
+        os: [ 'ubuntu-latest' ]
+        java-version: [ '11', '17' ]
```

Comment:
I'd say to add java 21 as well, since the cratedb matrix only contains nightly which uses 21.

Reply:
I am not sure it is available from `actions/setup-java` yet.

- https://github.com/actions/setup-java/pull/538
- https://github.com/actions/setup-java/issues/492

Do we need to switch to `oracle-actions/setup-java`, i.e. Oracle Java?

- https://github.com/oracle-actions/setup-java

Reply:
I've set up GH-115 to investigate updating to Java 21.

Reply:
GH-115 worked well, so I've merged it into this PR.